README.md

```diff
@@ -357,33 +357,38 @@ present, AI's operation version will include the value from this property.
 
 ## Using with Azure Functions
 
-Azure functions has out of the box integration with Application Insights, which automatically logs functions execution
+Azure functions has out of the box integration with Application Insights through ILogger, which automatically logs functions execution
```

Comment:
The integration will be via the `TelemetryClient` already registered in the service provider, so this might not be entirely correct; we might not need to make this change ☝️

Reply:
Yes, I think since you are now setting Log.Logger you might not need to use ILogger at all, which makes this change redundant.